files/en-us/web/accessibility/understanding_colors_and_luminance/index.html

```diff
@@ -2,289 +2,502 @@
+<img alt="0 to 100 Chroma Ramps, an example of high chroma vs no chroma." src="chroma-ramps.svg">
 
-<p>The "math" for the calculations is spelled out in WCAG 2.1 in the definition for <strong><a href="https://www.w3.org/TR/WCAG21/#dfn-relative-luminance">relative luminance</a></strong>, and is as follows: </p>
+<p>A good resource with many more detailed examples is at <a href="https://munsell.com/color-blog/difference-chroma-saturation/"> the Munsell color site</a>.</p>
 
-<p>For the sRGB colorspace, the relative luminance of a color is defined as L = 0.2126 * <strong>R</strong> + 0.7152 * <strong>G</strong>+ 0.0722 * <strong>B</strong> where <strong>R</strong>, <strong>G</strong> and <strong>B</strong> are defined as:</p>
+
+<h3 id="Color_Combinations">Color Combinations</h3>
+
+<p>Certain color combinations can be very problematic on a computer monitor or mobile device, and some color combinations can interfere with some impairments. Pure blue and red is one such example. But in addition, some impairments are deficient and can not differentiate all colors. And some colors, such as pure blue, are so low in luminance they need to be the darkest of two colors. Blue is also very low in resolution as there are far fewer blue cones, and they are scattered in our peripheral vision, and not present in our central vision. This leads to some color use guidelines:</p>
 
 <ul>
- <li>if RsRGB &lt;= 0.03928 then <strong>R</strong> = RsRGB/12.92 else <strong>R</strong> = ((RsRGB+0.055)/1.055) ^ 2.4</li>
- <li>if GsRGB &lt;= 0.03928 then <strong>G</strong> = GsRGB/12.92 else <strong>G</strong> = ((GsRGB+0.055)/1.055) ^ 2.4</li>
- <li>if BsRGB &lt;= 0.03928 then <strong>B</strong> = BsRGB/12.92 else <strong>B</strong> = ((BsRGB+0.055)/1.055) ^ 2.4</li>
+  <li>Never rely on hue alone for differentiating details. Adequate luminance contrast is required.</li>
+  <li>The green in a monitor makes up the vast majority of luminance (light), so it will usually be a significant part of the lighter colors.</li>
 </ul>
 
-<p>and RsRGB, GsRGB, and BsRGB are defined as:</p>
+<h3 id="Working_with_Blue">Working with Blue</h3>
 
 <ul>
- <li>RsRGB = R8bit/255</li>
- <li>GsRGB = G8bit/255</li>
- <li>BsRGB = B8bit/255</li>
+  <li>Pure blues should typically be the darkest of two colors</li>
+
+  <li>To use blue as the lighter of two colors, green needs to be added, otherwise there will be insufficient contrast and poor readability.</li>
+
+  <li>This is both due to the low contrast and due to the human eye seeing blue at a lower resolution than green and red.</li>
```

Comment:
```suggestion
  <li>Pure blues should typically be the darkest of two colors.</li>
  <li>To use blue as the lighter of two colors, green needs to be added, otherwise there will be insufficient contrast and poor readability.</li>
  <li>This is both due to the low contrast and due to the human eye seeing blue at a lower resolution than green and red.</li>
```

Removing newlines between list items.